realm-transformer/src/main/groovy/io/realm/transformer/RealmTransformer.groovy

```diff
@@ -278,4 +288,23 @@ class RealmTransformer extends Transform {
         return merged;
     }
 
+    // Use reflection to get the AndroidBuild from Android plugin for getting the path to android.jar
+    // There is no official way to get the path to android.jar for transform.
```

Comment:
Is there a bug for this on `b.android.com`? If not, we should create one and reference it here as well.